earthaccess/results.py

```diff
@@ -228,7 +228,9 @@ def __repr__(self) -> str:
         Temporal coverage: {self['umm']['TemporalExtent']}
         Size(MB): {self.size()}
         Data: {data_links}\n\n
-        """.strip().replace("  ", "")
+        """.strip().replace(
```

Comment:
Black likes this, Ruff does not, which one is right? @mfisher87

Reply:
I think we should pick one :) Ruff offers a formatter that re-implements Black, but I've never worked with it before. Personally, I'd turn Ruff formatting off and stick with Black due to my lack of experience, but if I had more time I'd try to learn more and switch to Ruff for everything.